app/javascript/packs/application.js

```diff
@@ -3,12 +3,15 @@
 
 import 'jquery/dist/jquery'
 import 'rails-ujs/lib/assets/compiled/rails-ujs'
-import 'jquery-ui/ui/widget'
+// import 'jquery-ui/ui/core'
+// import 'jquery-ui/ui/widgets/sortable'
 import 'popper.js/dist/umd/popper.min'
 import 'bootstrap/dist/js/bootstrap'
 import 'clipboard/dist/clipboard.min'
 import '@fortawesome/fontawesome-free/js/all'
-import 'jquery-ui/ui/widgets/sortable'
+// import 'sortablejs/modular/sortable.complete.esm'
+import './Sortable.min'
```

Comment:
GitHubのREADME見る感じだと下記のように書いてありますね。
https://github.com/SortableJS/Sortable

```suggestion
import Sortable from 'sortablejs';
```

Reply:
![17-43_37](https://user-images.githubusercontent.com/46378023/86450886-26db5680-bd55-11ea-8295-5f9ede679f58.jpg)
なぜかこんな感じの表示がでてきます。

Reply:
これはvscodeから見えてないので警告が出てるのかと。実際に起動してら正常にコンパイルされませんか？